web_client/views/imageViewerWidget/leaflet.js

```diff
@@ -4,8 +4,15 @@ var LeafletImageViewerWidget = ImageViewerWidget.extend({
     initialize: function (settings) {
         ImageViewerWidget.prototype.initialize.call(this, settings);
 
-        $('head').prepend(
-            $('<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.css">'));
+        if (!$('#large_image-leaflet-css', 'head').length) {
```

Comment:
Purely out of curiosity, is there a reason you prefer the jquery selector of `$('#large_image-leaflet-css', 'head')` over `$('head #large_image-leaflet-css')`?